presto-clickhouse/pom.xml

```diff
@@ -260,5 +260,24 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>test-clickhouse-mixedcase-integration</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-surefire-plugin</artifactId>
+                        <configuration>
+                            <includes>
+                                <include>**/TestClickhouseIntegrationMixedCase.java</include>
+                            </includes>
+                        </configuration>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
```

Comment:
@nishithakbhaskaran Why are we adding a new profile here?
I see there is an existing `ci` profile which includes all the tests [here](https://github.com/prestodb/presto/blob/master/presto-clickhouse/pom.xml#L228). Why don't we run that in CI instead?
Also I think, you can have a separate commit for enabling `ci` tests in here.